docs/Spatial.md

```diff
@@ -1,6 +1,99 @@
 
-[Basic spatial example](https://github.com/RIAEvangelist/node-phidget-API/blob/master/examples/spatial.js)
+##Methods
 
-ToDo : map out module and describe use
\ No newline at end of file
+The below information lists the parameters, events, and give you further information about your __Phidgets Spatial__. For low level development, [see here](https://github.com/RIAEvangelist/node-phidget-API/blob/master/docs/Phidget.md#connecting--phidgetparams).
+
+|Method call|Parameters|Description|
+|---|---|---|
+|[__phidget__.connect](https://github.com/RIAEvangelist/node-phidget-API/blob/master/docs/Phidget.md)|[__phidget__.params object](https://github.com/RIAEvangelist/node-phidget-API/blob/master/docs/Phidget.md#connecting--phidgetparams)|This connects the Phidget Spatial|
+|[__phidget__.whenReady](https://github.com/RIAEvangelist/node-phidget-API/blob/master/docs/Phidget.md)|function |This executes a function when the Phidgets RFID is ready to be used.|
+|[__phidget__.quit](https://github.com/RIAEvangelist/node-phidget-API/blob/master/docs/Phidget.md)|N/A |This method requests a disconnect from the phidget board.  The disconnected event will be dispatched when the connection has been successfully disconnected. |
+|__phidget__.observeGyro|'gyro'|This [observes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/observe) for the gyroscope on the __Spatial__ board. |
+|__phidget__.observeAcceleration|'accel'|This [observes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/observe) for the acceleration sensors on the __Spatial__ board. |
+|__phidget__.observeMagneticField|'mag'|This [observes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/observe) for the magnetic sensors, needed for the compass feature, on the __Spatial__ board. |
+
+##Data
+ 
+|Key|Data Type|Writable|Description|
+|---|---|---|---|
+|type|string|no|This is an identifier, Phidget GPS will always return 'PhidgetSpatial'|
+|accelerationMin|number|no|This returns the maximum negative acceleration that the desired axis will report.|
+|accelerationMax|number|no|This returns the maximum acceleration that the desired axis will report.|
+|angularRateMax|number|no|This returns the maximum angular momentum that the desired axis will report.|
+|angularRateMin|number|no|This returns the maximum angular momentum that the desired axis will report. |
+|dataRate|number|yes|This controls and reports current data rate at which data is being handled. In depth guide [here](http://www.phidgets.com/docs/1044_User_Guide) |
```

Comment:
Our documentation should not link to the external documentation for one single product.